backend/app/cmd/cmd.go

```diff
@@ -31,6 +31,20 @@ type CommonOpts struct {
 	Revision     string
 }
 
+// AdminPasswordOpt is shared among many commands for setting the server password
+// and then using it for backup/restore and such
+type AdminPasswordOpt struct {
```

Comment:
This one seems like a little bit too much inderection. I would rather put it to SupportCmdOpts and keep it as a "naked" (direct) option in the primary server opts